plugins/meta/flannel/flannel.go

```diff
@@ -202,6 +204,10 @@ func cmdAdd(args *skel.CmdArgs) error {
 		}
 	}
 
+	if runtime.GOOS == "windows" {
```

Comment:
Suggestion: both of these functions generate some kind of transformed configuration. So, can you structure the function something like:

```
func cmdAdd() {
// preamble
switch runtime.GOOS {
case `"windows":`
    buildWindowsDelegate()
case "linux":
   buildLinuxDelegate()
default:
   return err..
}
return delegateAdd()
}
```

This would be easier to test on the Linux side too.

Reply:
Yup - I made precisely this change to address the test comment :) will commit after adding a test